scripts/setup_k8s.sh

```diff
@@ -0,0 +1,17 @@
+# !/usr/bin/bash
+
+set -e -x
+
+yarn assets
+# Update filenames of generated CSS to be fingerprinted with matching JS fingerprint
+for file in public/assets/*.js
```

Comment:
This winds up looping over all the fingerprinted JS files, and if there is a corresponding CSS file, to fingerprint it with the same hash. Since we don't manage CSS with webpack, this seems like the best we can do for now.